php/libraries/NDB_Page.class.inc

```diff
@@ -194,6 +200,8 @@ class NDB_Page implements RequestHandlerInterface
 
     /**
      * Wrapper to create a static label to the current page
+     * FIXME: Setting the name to null is not a good idea because the name
```

Comment:
I didn't implement this change (like I did for addHeader), because when I made the change it affected some other unit tests in NDB_BVL_Instrument_ToJSON_Test and because it might affect other places in the backend as well.